app/assets/javascripts/edit_work_utils.js

```diff
@@ -501,6 +501,18 @@ $(() => {
     }
   });
 
+  // Fetch information for funder via the ROR API
+  $(document).on('change', '.ror-input', (el) => {
+    const $target = $(el.target);
+    const ror = $target.val().trim();
+    fetch(`https://api.ror.org/organizations/${ror}`).then()
```

Comment:
Might it be possible to please request error handling? I can create another issue if you wish to have this merged first (I have no preference at all, and do not mean this to sound critical).

```javascript
fetch(`https://api.ror.org/organizations/${ror}`).then()
      .then((response) => response.json())
      .then((responseJson) => {
        const { name } = responseJson;
        $target.closest('tr').find('.ror-output').val(name);
      })
      .catch((error) => {
        console.error('Error querying the Research Organization Registry (ROR) API:', error);
      });
});
```

Reply:
will do!

Reply:
... actually, this doesn't do what you want. Exceptions won't fall through to the end of the chain, so this would only handle an error in the jquery, not in the http, or in parsing the JSON. I'll file an issue for follow-up, and reference this.